src/PrestaShopBundle/Resources/views/Admin/Sell/CustomerService/MerchandiseReturn/Blocks/product.html.twig

```diff
@@ -0,0 +1,78 @@
+{#**
+ * 2007-2020 PrestaShop SA and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2020 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% set rowIsDisplayed =  (productIndex is defined and paginationNum is defined and productIndex > paginationNum) %}
+
+<tr id="merchandiseReturnProduct_{{ product.orderDetailId }}" class="cellProduct{% if rowIsDisplayed %} d-none d-print-table-row{% endif %}">
+  <td class="cellProductName">
+    <a href="{{ path('admin_product_form', {'id': product.productId.value }) }}">
+      <p class="mb-0 productName">{{ product.name }}</p>
+        <p class="mb-0 productReference">
+          {{ 'Reference number:'|trans({}, 'Admin.Orderscustomers.Feature') }}
+          {{ product.reference }}
+        </p>
+    </a>
+  </td>
+  <td class="cellProductQuantity">
+    {% if product.quantity > 1 %}
+      <span class="badge badge-secondary rounded-circle">{{ product.quantity }}</span>
+    {% else %}
+      {{ product.quantity }}
+    {% endif %}
+  </td>
+  <td class="text-right cellProductActions">
+    <button
+      type="button"
+      class="btn tooltip-link js-merchandise-return-product-delete-btn"
+      data-toggle="pstooltip"
+      data-placement="top"
+      data-merchandise-return-id="{{ editableMerchandiseReturn.merchandiseReturnId.getValue }}"
+      data-merchandise-return-detail-id="{{ product.orderDetailId }}"
+      data-delete-message="{{ 'Are you sure?'|trans({}, 'Admin.Notifications.Warning') }}"
+      data-original-title="{{ 'Delete'|trans({}, 'Admin.Actions') }}">
+      <i class="material-icons">delete</i>
+    </button>
+  </td>
+</tr>
+{% if product.customizations is not null %}
```

Comment:
Did you use Order page html for product listings as an inspiration ?

And if yes ... do you think we would be able to reuse them ? Else we will copy-paste this markup everywhere we display product listings, right ?

Reply:
Yeah for product list order page was primary inspiration, however this page has way less info overall. I think it could be reusable, if we add configuration to it which fields to use.